inst/validation/bbr-requirements.yaml

```diff
@@ -1034,6 +1034,10 @@ UTL-R013:
   description: wait_for_nonmem() doesn't error out if no stop time found
   tests:
   - BBR-UTL-013
+UTL-R014:
+  description: check_bbi_args() provides warning when `threads > 1` and parallel is `FALSE`
+  tests:
+  - BBR-UTL-014
```

Comment:
I know all the tests basically now confirm that parallel is TRUE (in submit_model and the workflow tests, but I think we need an additional recursive test that specifically confirms:

- if threads = 1 and parallel not set: parallel is untouched (test with parallel = TRUE and FALSE)

So far we have:
- if threads >1 and parallel set to FALSE: parallel is untouched and warning is rendered
- if threads > 1 and parallel not set: parallel is set to TRUE (see comment below for this one)

Reply:
> I noticed that none of the UTL-RXXX requirements have been added to any story. @kyleam do you know if thats by design?

Good question.  I don't spot these in the previous validation docs, which I think explains why they didn't get wired up by the mass import in https://github.com/metrumresearchgroup/bbr/pull/493

I suspect the answer is that, while these things are good things to test, they're not really conceptually tied to any stories.  I'm curious what @seth127's opinion is on this, but my view is that 1) we should test what we feel appropriate to test without worrying about whether it ties into story and 2) we shouldn't force a story just because there is a test.  As a consequence with both of those, I'm okay having tests that aren't linked, though in that case it would probably make sense to 1) drop the req and 2) drop the test ID.

Reply:
@kyleam I talked to Seth offline and he said that it was fine that the previous requirements weren't added to stories, but any _future_ requirements should be. I agree in terms of linking **tests** to stories, but I was under the impression that all **requirements** should be linked to a story (felt like that was important when rendering the docs - like it would fail when trying to map a requirement to a non-existent story). 


Based on my discussion with Seth though, it made sense to add this new requirement to a `submit_model` story, and leave the previous `UTL-RXXX` requirements alone. @rymarinelli After you make the requirement more general, can you add it to a story?

Reply:
> @kyleam I talked to Seth offline and he said that it was fine that the previous requirements weren't added to stories, but any _future_ requirements should be.

The fact that there are requirements for these at all is just an artifact of import process in the pr I linked above (gh-493).  I don't see a reason to keep them around.

> I agree in terms of linking **tests** to stories, but I was under the impression that all **requirements** should be linked to a story

In my reply, I wasn't suggesting to add requirements that aren't linked to a story (as I said, "it would probably make sense to 1) drop the req and 2) drop the test ID.").  So, I guess we're in agreement, but I'm not sure I understand what you're trying to say.

> (felt like that was important when rendering the docs - like it would fail when trying to map a requirement to a non-existent story. Assuming thats not the case though).

Again, I'm not saying we should add orphan reqs.  But if there are, no, things shouldn't fail altogether.  You can find the missing links with these helpers: https://metrumresearchgroup.github.io/mrgvalidate/reference/find_missing.html

Reply:
Oh no I wasnt suggesting you were saying anything lol, was just relaying my previous assumptions/concerns, mainly about issues with doc generation via mrgvalidate. Good to know about those helpers though, that definitely removes any concerns I had. 

But yeah im all for removing the other requirements in this PR. I guess it would make sense to remove the corresponding test ids too, but im also ok to leave them (can see reasons to remove and keep), but ill defer to you for that

Reply:
> But yeah im all for removing the other requirements in this PR.

Yeah, it is conceptually unrelated, so could be done elsewhere, but the main argument I see for doing it in this PR is that, assuming you're adding a requirement with the same stem, it avoids the val docs starting off at (... goes and greps ...) 015 while 001-014 are missing from the docs.